service/Dockerfile

```diff
@@ -2,10 +2,12 @@
 FROM openjdk:8-jre
 
 WORKDIR /usr/local/src
-COPY . /usr/local/src
+COPY . /usr/local/src/
```

Comment:
This global `COPY` seems wasteful. I think that we should only be adding the JAR and any supporting files that are necessary (configuration, data, etc.).